twentytwentyone-blocks/block-templates/index.html

```diff
@@ -1,8 +1,38 @@
 <!-- wp:template-part {"slug":"header","theme":"twentytwentyone-blocks","align":"full", "tagName":"header","className":"site-header"} /-->
 
+<!-- wp:query {"queryId":3,"query":{"perPage":10,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
```

Comment:
You can set `1` as this is the first Query instance. 

```suggestion
<!-- wp:query {"queryId":1,"query":{"perPage":10,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
```

Reply:
Additionally if you skip the `perPage` property it will search for the reading options from site (Blog pages show at most).

Reply:
Great, I'll remove the perPage property then. 👍 Thanks!